### PR TITLE
Add issue template. Copied from Julialang/julia.

### PR DESCRIPTION
**Pull request by odow**
This should help steer people towards Discourse as their first port of call.

Ref https://github.com/JuliaLang/julia/blob/master/.github/ISSUE_TEMPLATE.md

### CODE REVIEW

**Comment by mlubin** [comment]
Looks good! Maybe add a point about the difference between a JuMP issue and a solver wrapper issue?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please search or post to our Discourse site: https://discourse.julialang.org/c/domain/opt. We use
+the GitHub issue tracker for bug reports and feature requests only.
+
+If the bug you're experiencing is solver-specific (e.g., it only happens when you use Gurobi),
+the issue is best raised at the relevent solver-wrapper repository. If you are unsure, you should
+raise the problem on Discourse.
+
+If you've confirmed that the behaviour is a bug, and it relates to JuMP, this is the right place.
```
Comment (mlubin) [inline .github/ISSUE_TEMPLATE.md]: It's our job to confirm if something it's a bug. Maybe say "if you are reasonably confident that ...". Also repeat that JuMP feature requests are valid issues.

Comment (mlubin) [inline .github/ISSUE_TEMPLATE.md]: Also be consistent with behavior vs behaviour :)

Comment (odow) [reply .github/ISSUE_TEMPLATE.md]: Muscle-memory...

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please search or post to our Discourse site: https://discourse.julialang.org/c/domain/opt. We use
+the GitHub issue tracker for bug reports and feature requests only.
+
+If the bug you're experiencing is solver-specific (e.g., it only happens when you use Gurobi),
+the issue is best raised at the relevent solver-wrapper repository. If you are unsure, you should
```
Comment (joaquimg) [inline .github/ISSUE_TEMPLATE.md]: Typo: relevant

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please search or post to our Discourse site: https://discourse.julialang.org/c/domain/opt. We use
+the GitHub issue tracker for bug reports and feature requests only.
+
+If the bug you're experiencing is solver-specific (e.g., it only happens when you use Gurobi),
+the issue is best raised at the relevant solver-wrapper repository. If you are unsure, you should
```
Comment (mlubin) [inline .github/ISSUE_TEMPLATE.md]: Clarify that `Gurobi.jl` would be the corresponding solver-wrapper repository.

**Pull request merged**